Adapter_Revit_UI/CRUD/Update.cs

```diff
@@ -48,12 +49,68 @@ public partial class RevitUIAdapter
         //    if (bhomObject is IView || bhomObject is oM.Adapters.Revit.Elements.Family || bhomObject is InstanceProperties)
         //        element.Name = bhomObject.Name;
 
+        //FOR REFERENCE, IF OBJECT IS REVITFILEPREVIEW:
+        //TODO: this value should come from adapter PushType?
+        //bool updateFamilies = true;
+        //FamilyLoadOptions familyLoadOptions = new FamilyLoadOptions(updateFamilies);
+        //if (document.LoadFamily(revitFilePreview.Path, out family))
+        //{
+        //    SetIdentifiers(bhomObject, family);
+        //    element = family;
+        //}
+        //}
+
         /***************************************************/
         /****              UpdateProperty               ****/
         /***************************************************/
 
         // This method used to be called from the UpdateProperty component
         // Now it is never called as it doesn't override anymore
         // This logic should be called from the Push component instead
+
+
+        /***************************************************/
+        /****              Private Classes              ****/
+        /***************************************************/
+
+        private class FamilyLoadOptions : IFamilyLoadOptions
+        {
+            private bool m_Update;
+
+            public FamilyLoadOptions(bool update)
+            {
+                this.m_Update = update;
+            }
+
+            public bool OnFamilyFound(bool familyInUse, out bool overwriteParameterValues)
+            {
+                if (m_Update)
+                {
+                    overwriteParameterValues = false;
+                    return false;
+
+                }
+
+                overwriteParameterValues = true;
+                return true;
+            }
+
+            public bool OnSharedFamilyFound(Family sharedFamily, bool familyInUse, out FamilySource source, out bool overwriteParameterValues)
+            {
+                if (m_Update)
+                {
+                    overwriteParameterValues = false;
+                    source = FamilySource.Project;
+                    return false;
+
+                }
+
+                overwriteParameterValues = true;
+                source = FamilySource.Family;
+                return true;
+            }
+        }
+
+        /***************************************************/
```

Comment:
I have no idea what this is doing, but it does not look very compliant 😐 
Any way we can separate the behaviour in the Engine and then make the class public and move it to the oM?
If this is out of the scope for this PR I'd create a separate issue

Reply:
The thing is that the whole `Update.cs` file is actually empty, not containing any CRUD-related stuff.

Reply:
This is out of scope of this issue - tbh I do not know what it is doing either 🐒 Will make sure to raise an issue before merging this PR.

Reply:
Raise #575

Reply:
@alelom atm there is no support for `Update`. However, there is another PR in progress that will cover it (#546). Long story short, _Update.cs_ will be cleaned up in the next PR.